index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta charset='utf-8'>
+  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
+  <title>Page Title</title>
+  <meta name='viewport' content='width=device-width, initial-scale=1'>
+  <link rel='stylesheet' type='text/css' media='screen' href='main.css'>
+  <script src='main.js'></script>
+</head>
+<body>
+  <h2 class="my-title">Hello Microverse!</h2>
```

Comment:
I notice you are using an HTML` <h2>` tag, you need to know HTML best practices like for example the `h2`, `h3` ... tags are used to mark "subtitles" (h2), "subtitles" (h3) ... A single `h1` tag is taken into account on each page (there can only be a single “headline” to a blog post). On the other hand, there can be several subtitles (therefore several tags h2, h3, h4… up to h6).